src/renderer/services/file-management-system/ChunkedFileReader.ts

```diff
@@ -5,6 +5,8 @@ import BatchedTaskQueue from "../../entities/BatchedTaskQueue";
 
 import Md5Hasher from "./Md5Hasher";
 
+const READ_STREAM_MAX_CHUNK_SIZE = 100 * 1000 * 1000; // 100 MB
```

Comment:
Is this experimental (it didn't crash your machine) or is there a reference source you can link to?  Might be smart to comment it as such either way.

Reply:
Experimental: 200MB didn't crash on my machine, so I thought 100MB should be safe (and also decently large). I'll write a comment